lowfat/02-Exo-001-lowfat.xml

```diff
@@ -133,6 +144,7 @@
                                  pos="verb"
                                  after=" "
                                  type="participle active"
+                                 transliteration="bāʾiym"
```

Comment:
Note here that not every word has an SIL gloss for some reason. The Nodes trees likewise lack these glosses: e.g., 
```xml
<m word="EXO 1:1!5" xml:id="o020010010052" morph="Vqrmpa" lang="H" lemma="935" after=" " pos="verb" stem="qal" type="participle active" gender="masculine" number="plural" state="absolute" english="came" mandarin="去" transliteration="bāʾiym">בָּאִ֖ים</m>
```